### PR TITLE
feat: python version of ElasticBeanstalk example

### PR DESCRIPTION
**Pull request by zanhsieh**
---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by zanhsieh** [comment]
@garnaat 
Could you take a look at this PR please? Thank you.

File: python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    Raises:
+        Exception: The JSON can't be decoded or a property is missing.
+
+    """
+    try:
+        # Get the user parameters which contain the stack, artifact and file settings
+        user_parameters = job_data['actionConfiguration']['configuration']['UserParameters']
+        decoded_parameters = json.loads(user_parameters)
+
+    except Exception as e:
```
Comment (garnaat) [inline python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py]: You are not using `e` anywhere.  Also, at the moment you are catching ALL exceptions here but only raising an Exception if one of your required parameters is missing.  All other exceptions will be swallowed here.  That doesn't seem right.

File: python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+        # with a helpful message.
+        raise Exception('Your UserParameters JSON must include the Green environment name')
+
+    if 'application' not in decoded_parameters:
+        # Validate that the artifact name is provided, otherwise fail the job
+        # with a helpful message.
+        raise Exception('Your UserParameters JSON must include the application name')
+
+
+    return decoded_parameters
```
Comment (garnaat) [inline python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py]: PEP8 - preceded by two blank lines, should only be 1.

File: python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+
+def describe_health(environmentName):
+    response = eb.describe_environment_health(
+    EnvironmentName=environmentName,
```
Comment (garnaat) [inline python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py]: indentation is not correct here.

File: python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+
+def describe_app_version(version, application):
+    response = eb.describe_application_versions(
+        ApplicationName=application,
+        VersionLabels=[
+            version,
+        ]
+    )
+    print ("App Version Status: ", response['ApplicationVersions'][0]['Status'])
```
Comment (garnaat) [inline python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py]: No space between `print` and '(`

Several of these below

File: python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+    """
+    Creates the application Version to be deployed
+    """
+    bucket = artifact['location']['s3Location']['bucketName']
+    key = artifact['location']['s3Location']['objectKey']
+    version = artifact['revision'] + key
+    version = version.replace("test/MyApp/", "")
+    print ("version: ", version)
+
+    response = eb.create_application_version(
```
Comment (garnaat) [inline python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py]: `response` is never used

File: python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+    """
+    Deploys to Blue
+    """
+
+    response = eb.update_environment(
+        ApplicationName=application,
+        EnvironmentName=blueEnvironment,
+        VersionLabel=versionLabel,
+    )
+    print ("Blue Deploy result: ",response)
```
Comment (garnaat) [inline python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py]: PEP8 - no space between `print` and `(` and also need a space between `,` and `response`. 

Several of these below.

File: python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+            versionLabel = create_app_version(artifacts[0], application)
+            update_blue_env(blueEnvironment, versionLabel, application)
+            swap_blue_green(blueEnvironment, greenEnvironment)
+            put_job_success(job_id, 'Done')
+        else:
+            put_job_failure(job_id, 'Environment is not in Ready state')
+
+
+
+    except Exception as e:
```
Comment (garnaat) [inline python/elasticbeanstalk/elasticbeanstalk-bg-pipeline/resources/blue_green.py]: PEP8 - too many blank lines before this line.

**Comment by zanhsieh** [comment]
@garnaat
Done. Also have checked with elasticbeanstalk-environment. Please check.

**Comment by kaizio** [comment]
This needs to be updated to CDK v2.

**Pull request closed without merging**